src/block-components/image/edit.js

```diff
@@ -150,6 +151,28 @@ const Controls = props => {
 					} )
 				} }
 			/>
+			<AdvancedSelectControl
+				label={ __( 'Aspect Ratio', i18n ) }
+				attribute="imageAspectRatio"
+				options={ [
+					{ label: __( 'Original', i18n ), value: '' },
+					{ label: __( 'Square 1:1', i18n ), value: '1/1' },
+					{ label: __( 'Standard 4:3', i18n ), value: '4/3' },
+					{ label: __( 'Classic 3:2', i18n ), value: '3/2' },
+					{ label: __( 'Wide 16:9', i18n ), value: '16/9' },
+					{ label: __( 'Cinematic 2:1', i18n ), value: '2/1' },
+					{ label: __( 'Ultra Wide 3:1', i18n ), value: '3/1' },
+					{ label: __( 'Panoramic 4:1', i18n ), value: '4/1' },
+					{ label: __( 'Portrait 3:4', i18n ), value: '3/4' },
+					{ label: __( 'Classic Portrait 2:3', i18n ), value: '2/3' },
+					{ label: __( 'Tall 9:16', i18n ), value: '9/16' },
+				] }
+				helpTooltip={ {
+					title: __( 'Aspect Ratio', i18n ),
+					description: __( 'Sets the aspect ratio of the image', i18n ),
+				} }
```

Comment:
No need for this since it's quite self explanatory. We want as least amount of text as possible because every text would need to be translated to other languages.
```suggestion
```